SwiftyJSONAPI/JSONAPIError.swift

```diff
@@ -9,32 +9,32 @@
 import Foundation
 
 
-public class JSONAPIError: JSONPrinter {
+open class JSONAPIError: JSONPrinter {
```

Comment:
"Public" to "Open" was done by the conversion tool. In this case, I think is fine to let future generations to run free and subclass any class. Unless we want to start rethinking the architecture of the project, which is not the intention of this PR.